src/utils/index.js

```diff
@@ -1,24 +1,51 @@
 import client from '../client';
+import { getCommandsString } from '../../utils/index';
+
+/**
+ * Handles bot-specific commands
+ * @param {Object.<string, Object.<string, any>>} subCommands 
+ * @param {Object.<string, string>} strings 
+ */
+export const commandHandler = (subCommands, strings) => {
+  let cmd = parseCommandString();
+  let subCommand = cmd.subCommand;
+
+  if (!subCommand) {
+    client.message.channel.send(getCommandsString());
+    return;
+  }
+
+  let targetCmd = subCommands[subCommand];
+
+  if (!targetCmd) {
+    client.message.channel.send(strings.invalidSubCommand);
```

Comment:
one thing I did change was make this one string. Better one with nl than 2 pings of server

Reply:
yeah, I had noticed and updated